src/en/unsubscribe/unsubscribe.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Unsubscribe
+layout: 'layouts/base.njk'
+eleventyExcludeFromCollections: true
```

Comment:
We should add a translationKey to the first unsubscribe page so people can switch the language if need be.

```suggestion
translationKey: 'unsubscribe'
eleventyExcludeFromCollections: true
```